### PR TITLE
Add dependencies required to render line charts

### PR DESCRIPTION
**Pull request by anthonydillon**
## To QA
- Run `npm i` in this repo
- Run `make d3-min.js` to build d3.js and d3-min.js
- Checkout the GUI with the ISV profile charting [1]
- Replace the d3.js and d3-min.js files from this repo into your GUI branch at `jujugui/static/gui/src/app/assets/javascripts/`
- Clean and run the GUI `make clean-gui fast-babel run`
- Once running go to `http://0.0.0.0:6543/isv/:flags:/blues/` and check the chart is rendered

[1] https://github.com/juju/juju-gui/pull/1951/

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 	  node_modules/d3/src/selection/* \
 	  node_modules/d3/src/behavior/* \
 	  node_modules/d3/src/geom/hull.js \
 	  node_modules/d3/src/geom/polygon.js \
 	  node_modules/d3/src/scale/linear.js \
 	  node_modules/d3/src/scale/log.js \
+	  node_modules/d3/src/time/format.js \
+	  node_modules/d3/src/time/scale.js \
 	  node_modules/d3/src/transition/* \
+		node_modules/d3/src/svg/axis.js \
```
Comment (hatched) [inline Makefile]: indentation?

**Comment by anthonydillon** [comment]
@hatched sorry, fixed.

**Comment by kadams54** [comment]
QA is :ok:, code looks :+1:. Thanks @anthonydillon!

**Comment by anthonydillon** [comment]
Thanks, how do I merge this pull request? @makyo

**Comment by hatched** [comment]
It looks like something broke in Github and it merged the commits but didn't close the PR...

**Pull request closed without merging**